### ISSUE TITLE
Support for OctoPrint on Adventurer 5M

### ISSUE THREAD

**Issue by JStudios6118** [FlashforgeOfficial/AD5M_Series_Klipper#6]
I would like to connect OctoPrint to an Adventurer 5m using a Raspberry Pi 4 but it is not natively supported and OctoPi is unable to detect the printer. All plugins to fix this are no longer updated by their creators. I don't think there is anything I need to attach so I will leave it at this.

**Comment by MoonDoggie97** [comment]
+1 Would love to use octoprint on a pi as well connected to klipper

**Comment by jvarrone** [comment]
+1 also

**Comment by thhdragon** [comment]
pretty embarrassing even bambu printers have octoprint support and we dont even get a klipper build that BUILDS.

**Comment by visteras** [comment]
+1

**Comment by HunterAhlquist** [comment]
does FF det up these repos just to keep up appearances? I never see anybody from FF interact with the community. Makes me want to switch to another brand.

**Comment by MACCAMENZIES** [comment]
No. Flashforge uses proprietary software , a locked version of klipper.

They have no interest in groups like this I would imagine.
